### PR TITLE
1814 project air page

### PR DESCRIPTION
**Pull request by Kicksta**
add project AIR page and allow for pdf upload

### CODE REVIEW

**Comment by Kicksta** [comment]
If merged, I would also need permission for the ProjectAirForm: permission_required = f"{ProjectAirFiles._meta.app_label}.add_{ProjectAirFiles._meta.model_name}"
@jmsmkn, I understood you are the one to provide that?

File: app/grandchallenge/products/views.py [hunk 1]
```diff
@@ -191,6 +196,17 @@ class ContactPage(TemplateView):
...
 
+class ProjectAirPage(ListView):
+    template_name = "products/project_air.html"
+    model = ProjectAirFiles
+    context_object_name = "project_air_files"
+    queryset = ProjectAirFiles.objects.all()
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        return context
```
Comment (jmsmkn) [inline app/grandchallenge/products/views.py:207]: This doesn't do anything and can be removed

File: app/grandchallenge/products/templates/products/partials/project_air_description.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+</p>
+<h2 class="h4">Objective</h2>
+<p>The aim of Project AIR is to increase transparency in the market of AI for radiology. Project AIR develops a physical and digital infrastructure to perform validation studies demonstrating the stand-alone performance of commercially available (CE marked <i class="fas fa-info-circle text-radboud" data-toggle="tooltip" title={% include "products/partials/tooltip_CE.html" %}></i> or FDA cleared <i class="fas fa-info-circle text-radboud" data-toggle="tooltip" title={% include "products/partials/tooltip_FDA.html" %}></i>) AI based software for radiology use cases on a hidden, multicenter dataset. Readers studies are peformed to provide context to the results.
+<br><br>
+Would you like to join as a vendor or have feedback on the protocols? Please, send an email to <a href="{{'mailto:kicky.vanleeuwen@radboudumc.nl'}}" class="text-radboud">kicky.vanleeuwen@radboudumc.nl</a>.
```
Comment (jmsmkn) [inline app/grandchallenge/products/templates/products/partials/project_air_description.html:7]: The `{{'` and `'}}` can be removed.

File: app/grandchallenge/products/models.py [hunk 3]
```diff
@@ -152,3 +152,8 @@ class Meta:
 
     def get_absolute_url(self):
         return reverse("products:product-detail", kwargs={"slug": self.slug})
+
+
+class ProjectAirFiles(models.Model):
+    title = models.CharField(max_length=150)
+    study_file = models.FileField(upload_to=get_pdf_path)
```
Comment (jmsmkn) [inline app/grandchallenge/products/models.py:159]: This will make them all publicaly available, is that ok?

Comment (Kicksta) [reply app/grandchallenge/products/models.py:159]: they should be downloadable files, so yes.

File: app/grandchallenge/products/views.py (hunk 1 above)
Comment (Kicksta) [reply app/grandchallenge/products/views.py]: done

File: app/grandchallenge/products/templates/products/partials/project_air_description.html (hunk 2 above)
Comment (Kicksta) [reply app/grandchallenge/products/templates/products/partials/project_air_description.html]: done

**Comment by jmsmkn** [comment]
The tests are failing, you will need to merge or rebase from master and regenerate the migration here as there are multiple migrations with the same name (`Conflicting migrations detected; multiple leaf nodes in the migration graph: (0006_product_ce_under, 0006_projectairfiles in products)`). https://github.com/comic/grand-challenge.org/pull/1894/checks?check_run_id=2961572897

You will also need to fix the formatting which was found by the pre-commit hooks. Easiest thing to do is run the pre commit hooks locally and commit the result, or manually fix the following

```
Fix End of Files.........................................................Failed
- hook id: end-of-file-fixer
- exit code: 1
- files were modified by this hook

Fixing app/grandchallenge/products/templates/products/partials/project_air_files.html
Fixing app/grandchallenge/products/templates/products/partials/project_air_description.html

Mixed line ending........................................................Passed
Trim Trailing Whitespace.................................................Failed
- hook id: trailing-whitespace
- exit code: 1
- files were modified by this hook

Fixing app/grandchallenge/products/templates/products/partials/project_air_files.html
Fixing app/grandchallenge/products/templates/products/project_air.html
```
https://github.com/comic/grand-challenge.org/pull/1894/checks?check_run_id=2961572993

**Comment by Kicksta** [comment]
Thanks for the feedback. Will do. Shouldn't the formatting problems be caught by the following? Or am I missing something?
pyupgrade................................................................Passed
black....................................................................Passed
flake8...................................................................Passed
curlylint................................................................Passed

**Comment by jmsmkn** [comment]
No, those extra checks were added a few weeks ago but they are missing from what you're running as your branch is 80 commits behind master. Merge or rebase first and you should be fine.

**Review by jmsmkn — APPROVED** [review]

**Pull request merged**